lib/Transport.d.ts

```diff
@@ -66,31 +70,30 @@ export interface RequestEvent<T = any, C = any> {
 
 // ApiResponse and RequestEvent are the same thing
 // we are doing this for have more clear names
-export interface ApiResponse<T = any, C = any> extends RequestEvent<T, C> {}
+export interface ApiResponse<Response = any, Context = unknown> extends RequestEvent<Response, Context> {}
 
-declare type anyObject = {
-  [key: string]: any;
-};
+export type BodyType = Record<string, any> | string | Buffer | ReadableStream
+export type NDBodyType = Record<string, any>[] | string[] | Buffer | ReadableStream
```

Comment:
Don't see tests for `BodyType`: Buffer & ReadableStream & `NDBodyType`

Reply:
They are coming ;)
I'm fixing another issue that I've discovered while writing the test! :D